internal/pkg/config/config.go

```diff
@@ -48,6 +48,7 @@ type Params struct {
 	SiderolinkWireguardBindAddress       string `yaml:"siderolinkWireguardBindAddress"`
 	SiderolinkWireguardAdvertisedAddress string `yaml:"siderolinkWireguardAdvertisedAddress"`
 	SiderolinkDisableLastEndpoint        bool   `yaml:"siderolinkDisableLastEndpoint"`
+	SiderolinkUseGRPCTunnel              bool   `yaml:"siderolinkUseHTTPTunnel`
```

Comment:
since we are at it (consistent naming), maybe also
```suggestion
	SiderolinkUseGRPCTunnel              bool   `yaml:"siderolinkUseGRPCTunnel`
```
?

Reply:
Git is really playing some tricks on me right ow haha. Locally i have this, it says its pushed, but on github it's not there. I'll try a force push i guess

Reply:
that worked, that was a bit weird haha

Reply:
sorry, it was me force pushing :)
I've added your changes too now. Thanks